app/clients/hedera/hedera.go

```diff
@@ -1,3 +1,58 @@
 package hedera
 
-// TODO hedera client for interacting with hedera nodes
+import (
+	"encoding/json"
+	"fmt"
+	hederasdk "github.com/hashgraph/hedera-sdk-go"
+	"github.com/limechain/hedera-eth-bridge-validator/app/process/model/transaction"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type Client struct {
+	mirrorAPIAddress string
+	mirrorClient     *hederasdk.MirrorClient
+	httpClient       *http.Client
+}
+
+func (hedera Client) GetMirror() *hederasdk.MirrorClient {
```

Comment:
I'd not name the variable `hedera` a bit ambiguous - either go with the `client` or even short `c`